crates/xilem_masonry/src/view/flex.rs

```diff
@@ -142,6 +142,6 @@ impl ElementSplice for FlexSplice<'_> {
     }
 
     fn len(&self) -> usize {
-        self.ix
+        self.ix / 2
```

Comment:
Oooooh, you had to change the `len()` method as well?

I tried to do `self.ix += 2` at first, and I got crashes. I naively assumed that `self.ix += 1` would still work with spacers. Shame on me for not testing it worked ^^

Reply:
I don't think `len` used currently (IIRC it was previously only really used by the `List` ViewSequence, it's not yet clear whether it's really needed), but I think it's probably more correct that way, because the spacers are "invisible" views?

Weird that you got crashes with `self.ix += 2`. Do you know how you've got those crashes?